src/lib/util/apiResponseWrapper/apiResponseWrapper.ts

```diff
@@ -22,13 +24,22 @@ const getStatus = (statusCode: number): ApiResultStatus => {
     return ApiResultStatus.SUCCESS;
 };
 
-export type ApiResponseWrapper<T> = ApiResponseWrapperSuccess<T> | ApiResponseWrapperError<T>;
+export type ApiResponseWrapper<T> =
+    | ApiResponseWrapperSuccess<T>
+    | ApiResponseWrapperError<T>
+    | ApiResponseWrapperSuccessWithFile<T>;
 
 export type ApiResponseWrapperSuccess<T> = {
     isSuccess: true;
     result: T;
 };
 
+export type ApiResponseWrapperSuccessWithFile<T> = {
```

Comment:
wouldn't this  suffice: ApiResponseWrapperSucces<{ file: Blob, fileType: string}>